lib/curl_setup_once.h

```diff
@@ -374,6 +374,7 @@ typedef int sig_atomic_t;
  * Macro used to include assertion code only in debug builds.
  */
 
+#undef DEBUGASSERT
 #if defined(DEBUGBUILD) && defined(HAVE_ASSERT_H)
 #define DEBUGASSERT(x) assert(x)
```

Comment:
Where is DEBUGASSERT already defined?

Reply:
Many OS or library define DEBUGASSERT, you can see here:
https://github.com/search?l=C&p=1&q=DEBUGASSERT&type=Code
One I really hit is:
https://github.com/apache/incubator-nuttx/blob/master/include/assert.h
Anyway, it isn't harmful to undefine it before define.